artichoke-backend/src/extn/core/range/range.rb

```diff
@@ -22,10 +22,12 @@ def first(*args)
   def last(*args)
     return self.end if args.empty?
 
-    raise ArgumentError, "wrong number of arguments (given #{args.length}, expected 1)" unless args.length == 1
+    raise ArgumentError, "wrong number of arguments (given #{args.length}, expected #{inspect})" unless args.length == 1
```

Comment:
`inspect` prints debug information for the range object. I think we want the literal string `(0..1)`.